### PR TITLE
Unittest ag pulldown

### PR DESCRIPTION
**Pull request by sjanssen2**

### CODE REVIEW

**Comment by wasade** [comment]
Indirect execution, eg https://github.com/biocore/labadmin/blob/master/knimin/lib/data_access.py#L1048

**Comment by wasade** [comment]
sorry! wrong PR :)

File: knimin/tests/test_ag_pulldown.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+from tornado.escape import url_escape
+
+from knimin.tests.tornado_test_base import TestHandlerBase
+from knimin import db
+
+
+class testUpdateEBIStatusHandler(TestHandlerBase):
+    def test_get_not_authed(self):
+        os.environ["ASYNC_TEST_TIMEOUT"] = "30"
```
Comment (wasade) [inline knimin/tests/test_ag_pulldown.py]: would it make sense for this to be done once at the module level?

File: knimin/tests/test_ag_pulldown.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        response = self.multipart_post('/ag_pulldown/', data, files)
+        self.assertEqual(response.code, 200)
+        self.assertIn(('<h3 style="color:red">Pulldown Processing, please wait'
+                       ' for file download. It may take a while with many '
+                       'barcodes.</h3>'), response.body)
+
+
+class testAGPulldownDLHandler(TestHandlerBase):
+    def test_get_not_authed(self):
+        # os.environ["ASYNC_TEST_TIMEOUT"] = "30"
```
Comment (wasade) [inline knimin/tests/test_ag_pulldown.py]: can these comments be removed?

**Comment by wasade** [comment]
Two minor comments, :+1: otherwise

File: knimin/tests/test_ag_pulldown.py (hunk 1 above)
Comment (sjanssen2) [inline knimin/tests/test_ag_pulldown.py]: I don't know how :-/

**Pull request merged**